src/App.js

```diff
@@ -1,23 +1,18 @@
-import logo from './logo.svg';
 import './App.css';
+import {title} from "./title/title";
+import {GenreToggle} from "./genre-toggle/genre-toggle";
+import {SearchForm} from "./search-form/search-form";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+    <div className="app">
+      <header className="app-header">
+        { title }
       </header>
+      <main className="app-main">
+        <SearchForm></SearchForm>
```

Comment:
If we don't need to provide the child between tags we can use with closing tag  -  <SearchForm /> it's easier to read.

Reply:
Will be fixed in the next PR